docs/source/docker/getting_started/first_steps.rst

```diff
@@ -449,16 +429,28 @@ you might want to change:
     )
 
 
-**Checkpoints** from your training process will be stored in the output directory.
-You can use such a checkpoint in future worker runs by copying the checkpoint to
+**Checkpoints** from your training process will be stored in the Lightly API as artifacts.
```

Comment:
not sure if we should call this Lightly API or rather the Lightly Platform? Not sure that we use the word Lightly APi anywhere else, no?